tests/src/Kernel/Entity/SelectionHandlerTest.php

```diff
@@ -117,8 +117,6 @@ protected function setUp() {
    */
   public function testSelectionHandler() {
     $this->assertEquals(get_class($this->selectionHandler->getSelectionHandler()), 'Drupal\node\Plugin\EntityReferenceSelection\NodeSelection');
-    $this->assertEquals($this->selectionHandler->getConfiguration('handler'), 'default:node');
-    $this->assertEquals($this->selectionHandler->getConfiguration('target_type'), 'node');
```

Comment:
Yeah, which `$this->assertEquals(get_class($this->selectionHandler->getSelectionHandler()), 'Drupal\node\Plugin\EntityReferenceSelection\NodeSelection');` does I think. All that `$this->assertEquals($this->selectionHandler->getConfiguration('handler'), 'default:node');` test is that configuration is set on the selection handler. Or do you mean it is passed from the Og::getSelectionHandler method?